cortesa/reaction
# Cortesa patch 1 order table colum

Resolves no issue  
Impact: **minor**  
Type: **refactor**

## Issue
As a plugin developer I want to replace from my plugin the `orderTableColumn.js` but I can't because it is not registered as **Reaction Commerce** component.

## Solution
Refactor the component to be registered and change the parent component to use this new way.

## Breaking changes

## Testing
1. Create a custom plugin where you replace the component use `replaceComponent("OrderTableColumn", myCustomComponent, withMoment);`
2. Start the app.
3. Create and place an order
4. Go to order section (gear icon in right dashboard).
5. See the changes in the table shown

More detail for what each of these sections should include are available in our [Contributing Docs](https://docs.reactioncommerce.com/reaction-docs/master/contributing-to-reaction)
